src/main/java/by/onliner/cucumber/steps/ShoppingCartSteps.java

```diff
@@ -49,4 +51,52 @@ public void checkedEmptyCartMessage(String expectedMessage) {
                 String.format("The message [%s] wasn't contains at expected empty cart message [%s]",
                         emptyCartMassage, expectedMessage));
     }
+
+    @When("Get item name")
+    public void getItemName() {
+        ScenarioContext.getScenarioContext().setContext(Context.NAME, catalogPageItem.getItemName());
+    }
+
+
+    @Then("Item name from shopping cart must match the chosen item from catalog")
+    public void itemNameFromShoppingCartMustMatchTheChosenItemFromCatalog() {
+        String itemNameFromCatalog = String.valueOf(ScenarioContext.getScenarioContext().getContext(Context.NAME));
+        String itemNameFromCart = shoppingCartPage.getItemName();
+        Assert.assertEquals(itemNameFromCatalog, itemNameFromCart, String.format("Item name [%s] don't match " +
+                "item name [%s] from shopping cart", itemNameFromCatalog, itemNameFromCart));
+    }
+
+    @Then("[Перейти к оформлению] button should be visible")
+    public void isCompleteOrderButtonVisible() {
+        Assert.assertTrue(shoppingCartPage.isCompleteOrderButtonVisible(), "[Перейти к оформлению] button isn't " +
+                "visible");
+    }
+
+    @When("Get item price")
+    public void getItemPrice() {
+        ScenarioContext.getScenarioContext().setContext(Context.PRICE, shoppingCartPage.getPrice());
+    }
+
+
+    @And("Click + button")
+    public void clickPlusButton() {
+        shoppingCartPage.clickQuantityInputPlusButton();
+    }
+
+
+    @Then("The number from quantity input will match [{int}]")
+    public void theNumberFromQuantityInputWillMatch(int expectedNumber) {
+        int numberFromQuantityInput = shoppingCartPage.getNumberFromQuantityInput();
+        Assert.assertEquals(numberFromQuantityInput, expectedNumber, String.format("The number [%d] from quantity input don't" +
+                " match [%d]",numberFromQuantityInput, expectedNumber));
+    }
+
+
+    @And("The price after adding the same item was match first price*{int}")
```

Comment:
Was? i think you meant 'should' or smth like that as you are validating that right now

Reply:
Thx, little mistake, fixed that